src/Nancy/Responses/DefaultJsonSerializer.cs

```diff
@@ -100,6 +100,7 @@ private static bool IsJsonType(string contentType)
             return contentMimeType.Equals("application/json", StringComparison.OrdinalIgnoreCase) ||
                 contentMimeType.StartsWith("application/json-", StringComparison.OrdinalIgnoreCase) ||
                 contentMimeType.Equals("text/json", StringComparison.OrdinalIgnoreCase) ||
+                contentMimeType.Equals("application/problem+json", StringComparison.OrdinalIgnoreCase) ||
                 (contentMimeType.StartsWith("application/vnd", StringComparison.OrdinalIgnoreCase) &&

```

Comment:
I think it's technically correct In terms of the spec (or at least convention) on MIME types - custom types should always start with `application/vnd` to indicate a vendor-specific type. I guess in terms of what a serializer cares about it's just a question of whether it's a JSON type rather than wether it's a valid MIME type. I'll look at changing this and adding some tests.